components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag

```diff
@@ -73,6 +73,31 @@ if (!user) {
         } else {
             result = 400;
         }
+    }else if (uriMatcher.match("/{context}/api/devices/sketch/generate_link")) {
```

Comment:
Need a space after the curly brace. Please format the code "} else if (...) {"

Reply:
This is fixed 0ce103e1ad33617f640d5c03b57477f2b5adb4db